lib/core/src/system/gpu/data/texture.rs

```diff
@@ -326,49 +331,72 @@ with_texture_format_relations!(generate_internal_format_instances []);
 
 
 
-// =====================
-// === TextureSource ===
-// =====================
+// =======================
+// === Texture sources ===
+// =======================
+
+// === Empty Texture ===
+
+/// An empty texture, created as 1x1 texture in Rgba format.
+pub struct EmptyTexture;
```

Comment:
This is not empty texture. It is an UnitTexture. Empty texture IMO means that you create a texture but does not initialize it with data (e.g. for the needs of framebuffers). Let's talk about it via phone tomorrow.